site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -80,7 +80,15 @@ gcloud compute firewall-rules create game-server-firewall \
 
 ## Creating the cluster
 
-A [cluster][cluster] consists of at least one *control plane* machine and multiple worker machines called *nodes*. In Google Kubernetes Engine, nodes are [Compute Engine virtual machine][vms] instances that run the Kubernetes processes necessary to make them part of the cluster.
+A [cluster][cluster] consists of at least one *control plane* machine and multiple worker machines called *nodes*. In Google Kubernetes Engine, nodes are [Compute Engine virtual machine][vms] instances that run the Kubernetes processes necessary to make them part of the cluster. 
+
+{{% feature publishVersion="1.29.0" %}}
```

Comment:
Since this is going to get merged after 1.30.0 is released - no more need for feature shortcoding any of this.